cmd/ignite-spawn/ignite-spawn.go

```diff
@@ -47,7 +48,7 @@ func StartVM(vm *api.VM) error {
 	// Serve DHCP requests for those interfaces
 	// This function returns the available IP addresses that are being
 	// served over DHCP now
-	ipAddrs, err := container.StartDHCPServers(vm, dhcpIfaces)
+	_, err = container.StartDHCPServers(vm, dhcpIfaces)
```

Comment:
Please cleanup the code that returns this unnecessary slice

Reply:
Done in 872516a.